src/ServiceControl.Audit/Auditing/AuditPersister.cs

```diff
@@ -82,17 +90,29 @@ public async Task<IReadOnlyList<MessageContext>> Persist(List<MessageContext> co
                         {
                             Logger.Debug($"Adding audit message for bulk storage");
                         }
-                        await bulkInsert.StoreAsync(processedMessage).ConfigureAwait(false);
+
+                        using (auditBulkInsertDurationMeter.Measure())
```

Comment:
Given that we mostly only need this for internal purposes right now would it make sense to make this completely opt out from the hot path when not enabled? I think given the risk of CPU churn due to the metrics epoch refresh logic I would be in favor of that